### PR TITLE
Suite co2 emissions

### PR DESCRIPTION
**Pull request by Esteban100k**
CO2 graphs in Blockexplorer.

### CODE REVIEW

File: src/api/index.tsx [hunk 1]
```diff
@@ -211,3 +211,84 @@ export async function loadValidatorsInfo() {
             })
     })
 }
+
+export const fetchDailyEmissions = (dates: any) => {
+    return new Promise((resolve, reject) => {
+        console.log("datesDailyEmissions",dates);
```
Comment (Ayagoumi) [inline src/api/index.tsx]: Remove the log here.

File: src/app/components/Statistics/DateRange/DateRange.tsx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  InitianEndDate,
+  setEndDate,
+  setStartDate,
+  darkMode
+}) => {
+  const changeDate = date => {
+    setEndDate(new Date());
+    setStartDate(new Date(date));
+  };
+  console.log({
```
Comment (Ayagoumi) [inline src/app/components/Statistics/DateRange/DateRange.tsx]: also here

**Review by Ayagoumi — CHANGES_REQUESTED** [review]
Fix the conflicts in this PR

File: src/app/components/Statistics/DateRange/DateRange.tsx (hunk 2 above)
Comment (Esteban100k) [reply src/app/components/Statistics/DateRange/DateRange.tsx]: @Ayagoumi solved conflicts and removed logs

**Pull request closed without merging**